src/helpers/validation-shemas.js

```diff
@@ -0,0 +1,16 @@
+const joi = require('joi');
+
+exports.signUpSchema = joi.object().keys({
+  firstname: joi.string().alphanum().min(3).max(30)
+    .required(),
+  lastname: joi.string().alphanum().min(3).max(30)
```

Comment:
```firstname``` and ```lastname``` should be a string with alpha only .remove ```alphanum()``` from here.
and use regex with chars [a-z and A-Z]